modules/onetbb/2021.10.0/presubmit.yml

```
@@ -0,0 +1,13 @@
+matrix:
+  platform:
+    - centos7
+    - debian10
+    - ubuntu2004
+    - macos
+    - windows
+tasks:
+  verify_targets:
+    name: Verify build targets
+    platform: ${{ platform }}
+    build_targets:
+      - '@oneTBB//:tbb'
```

```suggestion
      - '@onetbb//:tbb'
```

Having a `onetbb` alias for `tbb` would make this easier to use (just `@onetbb`). Could be worth adding to upstream.